src/app/backend/deploy.go

```diff
@@ -86,13 +96,23 @@ func DeployApp(spec *AppDeploymentSpec, client *client.Client) error {
 		Labels:      labels,
 	}
 
+	containerSpec := api.Container{
+		Name:  spec.Name,
+		Image: spec.ContainerImage,
+	}
+
+	if spec.ContainerCommand != nil {
+		containerSpec.Command = []string{*spec.ContainerCommand}
+	}
+
+	if spec.ContainerCommandArgs != nil {
+		containerSpec.Args = []string{*spec.ContainerCommandArgs}
```

Comment:
For the command this makes sense but what about arguments?
When we provide multiple arguments will kubernetes manage it as only 1 element instead of multiple arguments split to multiple elements?